src/components/apps/launch/AppLaunchForm.js

```diff
@@ -48,100 +51,108 @@ import {
     constants.PARAM_TYPE.REFERENCE_GENOME,
     constants.PARAM_TYPE.REFERENCE_SEQUENCE,
     constants.PARAM_TYPE.REFERENCE_ANNOTATION,
 ];
 
-const StepContent = ({ id, hidden, step, label, children }) => (
-    <fieldset id={id} hidden={hidden}>
-        <legend>
-            <Typography variant="caption">
-                {getMessage("stepLabel", { values: { step: step + 1, label } })}
-            </Typography>
-        </legend>
-        {children}
-    </fieldset>
+const StepContent = ({ id, hidden, step, label, children, offsetHeight }) => (
+    <PageWrapper appBarHeight={offsetHeight}>
+        <fieldset id={id} hidden={hidden}>
+            <legend>
+                <Typography variant="caption">
+                    {getMessage("stepLabel", {
+                        values: { step: step + 1, label },
+                    })}
+                </Typography>
+            </legend>
+            {children}
+        </fieldset>
+    </PageWrapper>
 );
 
-const StepperBottomNavigation = ({
-    formId,
-    showSaveQuickLaunchButton,
-    showSubmitButton,
-    handleBack,
-    handleNext,
-    handleSaveQuickLaunch,
-    handleSubmit,
-}) => (
-    <BottomNavigation
-        showLabels
-        value={showSubmitButton ? "submit" : "next"}
-        onChange={(event, value) => {
-            switch (value) {
-                case "submit":
-                    handleSubmit(event);
-                    break;
-                case "saveQL":
-                    handleSaveQuickLaunch();
-                    break;
-                case "next":
-                    handleNext();
-                    break;
-                default:
-                    handleBack();
-                    break;
-            }
-        }}
-    >
-        <BottomNavigationAction
-            id={buildDebugId(formId, ids.BUTTONS.STEP_BACK)}
-            label={getMessage("back")}
-            value="back"
-            icon={<ArrowBack />}
-        />
-        {showSaveQuickLaunchButton ? (
-            <BottomNavigationAction
-                id={buildDebugId(formId, ids.BUTTONS.SAVE_AS_QUICK_LAUNCH)}
-                label={getMessage("saveAsQuickLaunch")}
-                value="saveQL"
-                icon={<Save />}
-            />
-        ) : (
-            <BottomNavigationAction disabled={true} />
-        )}
-        {showSubmitButton ? (
-            <BottomNavigationAction
-                id={buildDebugId(formId, ids.BUTTONS.SUBMIT)}
-                label={getMessage("launchAnalysis")}
-                value="submit"
-                icon={<PlayArrow />}
-            />
-        ) : (
-            <BottomNavigationAction
-                id={buildDebugId(formId, ids.BUTTONS.STEP_NEXT)}
-                label={getMessage("next")}
-                value="next"
-                icon={<ArrowForward />}
-            />
-        )}
-    </BottomNavigation>
-);
+const StepperBottomNavigation = React.forwardRef((props, ref) => {
```

Comment:
Needed to set `ref` for functional components. See https://github.com/reactjs/reactjs.org/issues/2120